src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -1238,6 +1249,25 @@ export default class InviteDialog extends React.PureComponent<IInviteDialogProps
         }
     }
 
+    async onLinkClick(e) {
```

Comment:
```suggestion
    public async onLinkClick(e) {
```

or whatever member access was intended.